server/lib/mappings/result-data.json

```diff
@@ -1,590 +1,189 @@
-                      }
-                    }
-                  }
-                }
-              },
-              "throughput": {
-                "properties": {
-                  "iops_sec": {
-                    "properties": {
-                      "uid": {
-                        "type": "string"
-                      },
-                      "role": {
-                        "type": "string"
-                      },
-                      "client_hostname": {
-                        "type": "string"
-                      },
-                      "mean": {
-                        "type": "double"
-                      },
-                      "description": {
-                        "type": "string"
-                      },
-                      "server_port": {
-                        "type": "string"
-                      },
-                      "closest sample": {
-                        "type": "long"
-                      },
-                      "stddev": {
-                        "type": "long"
-                      },
-                      "samples": {
-                        "properties": {
-                          "timeseries": {
-                            "properties": {
-                              "date": {
-                                "type": "long"
-                              },
-                              "read(0) or write(1)": {
-                                "type": "long"
-                              },
-                              "value": {
-                                "type": "double"
-                              }
-                            }
-                          },
-                          "value": {
-                            "type": "double"
-                          }
-                        }
-                      },
-                      "server_hostname": {
-                        "type": "string"
-                      },
-                      "stddevpct": {
-                        "type": "long"
-                      }
-                    }
-                  }
-                }
-              },
-              "parameters": {
+            "@timestamp": {
+                "type": "date",
+                "format": "dateOptionalTime"
+            },
+            "timestamp_original": {
+                "type": "long"
+            },
+            "@metadata": {
                 "properties": {
-                  "benchmark": {
-                    "properties": {
-                      "uid": {
-                        "type": "string"
-                      },
-                      "primary_metric": {
-                        "type": "string"
-                      },
-                      "benchmark_name": {
-                        "type": "string"
-                      },
-                      "max_stddevpct": {
-                        "type": "long"
-                      }
+                    "benchmark_name": {
```

Comment:
It seems we need to move some of this data to benchmark specific data sub-fields.

Reply:
Moved all "benchmark" data, that taken from the `parameters.benchmark[0]` array entry to the "benchmark" namespace.